daily_reports.rb

```diff
@@ -31,11 +31,13 @@
 require_relative './models/project_factory'
 
 def all_projects(date, slack, text, rerun, verbose, customer_facing)
-  ProjectFactory.new().all_projects_as_type.each do |project|
+  ProjectFactory.new().all_active_projects_as_type.each do |project|
     begin
       project.daily_report(date, slack, text, rerun, verbose, customer_facing)
-    rescue AzureApiError => e
+    rescue AzureApiError, AwsSdkError => e
+      puts "Generation of daily report for project #{project.name} stopped due to error: "
       puts e
+      puts "_" * 50
```

Comment:
Time for secret bash variables! Bash has it's own 'shell variables', which are bonus environment variables provided by the Bash shell. Instead of setting a hard value of `50` here, you could use the `ENV['COLUMNS']` variable to print as many underscores as there are character columns in the current terminal.

Reply:
Interesting! Though unfortunately doesn't work here (at least on my local machine) - don't think it warrants asking users to take extra steps to use it.